Code/Framework/AzToolsFramework/AzToolsFramework/aztoolsframework_files.cmake

```diff
@@ -615,6 +615,7 @@ set(FILES
     ComponentMode/ComponentModeCollection.cpp
     ComponentMode/ComponentModeDelegate.h
     ComponentMode/ComponentModeDelegate.cpp
+    ComponentMode/ComponentModeDelegateNotificationBus.h
```

Comment:
update: Could we please name this file `ComponentModeDelegateBus.h` - that way if we want to add a request bus in future (which might happen) we don't need another file. Most bus files follow this pattern so would be good to drop the 'Notification' part from the filename itself. Thanks!